Containerfile

```diff
@@ -62,6 +62,9 @@ RUN dnf install -y python3.9-pip && pip3 install -U pip
 
 RUN install-from-bindep && rm -rf /output/wheels
 
+# python -> python3 SymLink (as in Automation Platform EEs)
+RUN alternatives --install /usr/bin/python python /usr/bin/python3.9 209
```

Comment:
what does 209 mean here?

Reply:
Shouldn't be necessary. Just do:

```
alternatives --set python /usr/bin/python3.9
```

Reply:
@shanemcd I tested this in a clean `awx-ee` image. 
```bash
docker run --user=root -it quay.io/ansible/awx-ee:latest bash

bash-5.1$ alternatives --set python /usr/bin/python3.9
cannot access /var/lib/alternatives/python: No such file or directory
```

Seems you must `--install` before you can `--set`.


```bash
alternatives --install /usr/bin/python python /usr/bin/python3.9
```
Does not work either, because `<priority>` is a mandatory argument for `--install`.

---

Priority `209` was selected randomly, the value of this number is not consequential since we are only defining a single `alternative`.